package.json

```diff
@@ -0,0 +1,79 @@
+{
+  "name": "@strg-at/template-node",
+  "version": "0.1.0",
+  "description": "Nuxt application for MethMedia",
+  "scripts": {
+    "start": "nuxt start",
+    "dev": "nuxt dev --host 0.0.0.0",
+    "build": "nuxt build",
+    "preview": "nuxt preview ",
+    "generate": "nuxt generate",
+    "postinstall": "nuxt prepare",
+    "storybook": "storybook dev -p 6006",
+    "build-storybook": "storybook build",
+    "lint": "eslint . --ext .vue,.js,.jsx,.cjs,.mjs,.ts,.tsx,.cts,.mts --fix --ignore-path .gitignore",
+    "lint:fix": "eslint \"src/**/*.ts\" --ignore-pattern \"src/**/*.spec.ts\" --fix",
+    "test": "vitest run --environment jsdom",
+    "test:unit": "vitest run --environment jsdom",
+    "test:watch": "vitest dev --environment jsdom -u",
+    "test:coverage": "vitest --environment jsdom --coverage"
+  },
+  "dependencies": {
+    "@nuxtjs/eslint-config-typescript": "^12.0.0",
```

Comment:
this might be an oversight from the originating repo, all versions should be pinned for renovate I guess, so all carets should be removed here. btw. @jazzlyn @tyriis do we run renovate on our templates? If not we should consider reducing dependencies as much as possible here.

Reply:
also this shoud be a devDependency, as it is linting